src/nested_pandas/series/ext_array.py

```diff
@@ -755,6 +778,23 @@ def to_arrow_ext_array(self, list_struct: bool = False) -> ArrowExtensionArray:
             return ArrowExtensionArray(self._list_array)
         return ArrowExtensionArray(self._chunked_array)
 
+    def to_pyarrow_scalar(self, list_struct: bool = False) -> pa.ListScalar:
```

Comment:
Probably a good idea to add this to the docs here: https://github.com/lincc-frameworks/nested-pandas/blob/main/docs/reference/ext_array.rst